src/components/Class_example/Class_example.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '../Func_example/Func_example.css';
+
+class Class_example extends React.Component {
+    render() {
+        return (
+            <div>
+                <input className="main" placeholder={this.props.title}>
+                </input>
```

Comment:
You do not need this close element. 
https://react-cn.github.io/react/tips/self-closing-tag.html